addon/components/polaris-unstyled-link.js

```diff
@@ -44,6 +39,11 @@ export default Component.extend({
    */
   external: false,
 
+  /*
+   * Internal properties.
+   */
+  dataPolarisUnstyled: 'true',
```

Comment:
This is internal so shouldn't be up here amongst the public properties

Reply:
^^ @sivakumar-kailasam 

Perhaps this should get a JSDoc comment to flag it as `@private`, and we can drop the `Internal properties` comment (something we've been doing over the past few months already)?